src/sonic-bgpcfgd/bgpcfgd/runner.py

```diff
@@ -40,7 +40,8 @@ def add_manager(self, manager):
         db = swsscommon.SonicDBConfig.getDbId(db_name)
         if db not in self.db_connectors:
             if db_name == "CHASSIS_APP_DB":
-                self.db_connectors[db] = swsscommon.DBConnector(db_name, 0, False, '')
+                 # isTcpConn is True since we cannot use unix socket for Chassis DB connection from LC
```

Comment:
># isTcpConn is True since we cannot use unix socket for Chassis DB connection from LC [](http://example.com/codeflow?start=17&length=85)

Could you exactly revert the 2 files, not adding extra logic/comments?

Reply:
done